doc/api/process.md

```diff
@@ -2657,6 +2657,8 @@ The available scopes are:
 * `fs` - All File System
 * `fs.read` - File System read operations
 * `fs.write` - File System write operations
+* `child` - Child process spawning operations
+* `worker` - Worker thread spawning operation
```

Comment:
```suggestion
* `worker` - Worker thread spawning operations
```

Reply:
I wrote it that way because we currently support one operation that creates a Worker instance. Would it be better to use the plural?

Reply:
No, I thought it was a typo. I didn't know it was intentional.